README.md

```diff
@@ -36,6 +37,12 @@ The fields have to be one of the types that the sync package supports in order t
 - sync.Int64, allows for concurrent int64 manipulation
 - sync.Float64, allows for concurrent float64 manipulation
 - sync.Bool, allows for concurrent bool manipulation
+- sync.SecretString, same as `sync.String` but for secrets
+- sync.SecretInt64, same as `sync.Int64` but for secrets
+- sync.SecretFloat64, same as `sync.Float64` but for secrets
+- sync.SecretBool, same as `sync.Bool` but for secrets
```

Comment:
I think that only the `SecretString` is needed since I cannot recall having an integer as a secret ever. Even a 4 digit pin was always a string and not a number. SecretBool has a 50% chance to be guessed...

Reply:
You are right about `Bool`, but I wanted to be consistent for all types

Reply:
I think that the rest is not needed.
Maybe we just create the `sync.Secret` type which will be a string underlying.
What do you think?

Reply:
OK. I think that will work :+1: